.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,141 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+  "name": "Python 3",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  // TODO: Bullseye has glibc 2.31 and cadquery-ocp@7.7.2 needs 2.35
+  "image": "mcr.microsoft.com/devcontainers/python:1-3.10-bookworm",
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "ghcr.io/deep-space-cartel/devcontainers-features/apt:latest": {
+      // TODO: cmake is a dependency of Python package "pycairo"
+      // TODO: as well as those: libcairo2-dev pkg-config python3-dev // https://pycairo.readthedocs.io/en/latest/getting_started.html
+      // ffmpeg libsm6 libxext6 to fix ImportError: libGL.so.1: cannot open shared object file: No such file or directory
+      // openscad: is needed for "pc inspect :test"
+      "packages": "shellcheck bash-completion cmake libcairo2-dev pkg-config python3-dev ffmpeg libsm6 libxext6 openscad" // shfmt
+    },
+    "ghcr.io/deep-space-cartel/devcontainers-features/pip:1": {
+      "packages": "pre-commit poetry",
+      "location": "/opt/pip"
+    },
+    "ghcr.io/deep-space-cartel/devcontainers-features/starship:latest": {
+      "version": "latest"
+    }
```

Comment:
I started using https://starship.rs/ for my terminal and kinda happy with it, but I saw @openvmp is using something else like `tmux` or `zsh`. This is very opinionated topic what's the best. Here is the example of my prompt:

![image](https://github.com/user-attachments/assets/734d384e-0516-4577-bf44-88c76157c164)

Here is another example which in addition to git state shows:

- Current package version.
- Current Python version.
- Current Python environment.

![image](https://github.com/user-attachments/assets/c3f5ba79-fadb-4aa0-8817-d44de991bf8f)